.github/workflows/pull-request-ci.yaml

```diff
@@ -199,14 +199,22 @@ jobs:
           echo "::set-env name=SSH_AUTH_SOCK::${SSH_AUTH_SOCK}"
           echo "::set-env name=SSH_AGENT_PID::${SSH_AGENT_PID}"
 
-      # set GKE secret. TBD: use vault instead
+      - name: Query vault for GKE secrets
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          exportEnv: false
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
```

Comment:
We could also pin the CA cert of volt.cap.explore.suse.dev, using `caCertificate` (see https://github.com/marketplace/actions/vault-secrets), instead of using the trust chain of RootCert, IntermediateCert, VoltCert. But maybe it's more hassle than anything.

Reply:
PR rebased, conflicts resolved. Not doing anything with certs.